model.py

```diff
@@ -63,6 +63,9 @@ def is_supported(self, config):
             if not self.is_supported_mode(config):
                 return False, ("casual" if self.rated else "rated")
 
+            if self.challenger_name in config.get("block_list", []):
```

Comment:
It's better to use `config.get("block_list") or []` to make sure a list is the result. If a user has `block_list:` with nothing beneath it, the current expression will return `None`, leading to an error since `X in None` is not valid. Imagine that the user has this in their config.yml:
```yml
challenges:
  block_list:
    - user1
    - user2
```
Then, the user decides to delete the list.

```yml
challenges:
  block_list:
```
This will cause the error.